gobblin-example/src/main/java/gobblin/example/wikipedia/WikipediaExtractor.java

```diff
@@ -282,12 +388,12 @@ public JsonElement readRecord(@Deprecated JsonElement reuse) throws DataRecordEx
 
   @Override
   public long getExpectedRecordCount() {
-    return this.numRequestedTitles * this.revisionsCnt;
+    return 0;
```

Comment:
We don't know the expected record count because the revisions are not sequential, the only way to know is to actually consume revisions.